src/main/java/com/criteo/vips/Vips.java

```diff
@@ -80,13 +80,17 @@ private static boolean tryLoadLibrariesFromJar() throws IOException {
     }
 
     private static boolean isWindows() {
-        return SystemName.indexOf("win") >= 0;
+        return SYSTEM_NAME.contains("win");
```

Comment:
I've seen some code that would consider `os.name` to be "Darwin" on macOS. This doesn't seem to be true on my Macbook, but I'd prefer to avoid changing this.

Reply:
Now I realize that the initial code was potentially bugged as well 😓 Anyway, let's not touch it!

Reply:
Ok! I rolled back the change!

Reply:
Haha, Thought so. They are semantically identical ;)